vmdb/app/views/layouts/quadicon/_cim_base_storage_extent.html.haml

```diff
@@ -0,0 +1,2 @@
+.flobj
+  %img{:src => "/images/icons/#{size}/cim_base_storage_extent.png", :border => "0"}
```

Comment:
please, create a helper for this and replace all these 2 line partials with a call to this helper